### PR TITLE
docs: Fix contributing.md link in running-locally.md

### PR DESCRIPTION
**Pull request by shuheiktgw**
I just noticed running-locally.md's link pointed to a wrong location, so fixed it. Thanks for your review!

Checklist:

* [x] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [x] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [x] Optional. My organization is added to USERS.md.
* [x] I've signed the CLA and my build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

### CODE REVIEW

File: docs/developer-guide/running-locally.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
 During development, it might be viable to run ArgoCD outside of a Kubernetes cluster. This will greatly speed up development, as you don't have to constantly build, push and install new ArgoCD Docker images with your latest changes.
 
-You will still need a working Kubernetes cluster, as described in the [contributing.md](Contribution Guide), where ArgoCD will store all of its resources.
+You will still need a working Kubernetes cluster, as described in the [contributing.md](contributing.md), where ArgoCD will store all of its resources.
```
Comment (jannfis) [inline docs/developer-guide/running-locally.md:7]: Actually, I think the two should be exchanged - so `[contributing.md](Contribution Guide)` should be `[Contribution Guide](contributing.md)`

File: docs/developer-guide/running-locally.md [hunk 2]
```diff
@@ -4,9 +4,9 @@
...
 
-If you followed the [contributing.md](Contribution Guide) in setting up your toolchain, you can run ArgoCD locally with these simple steps:
+If you followed the [contributing.md](contributing.md) in setting up your toolchain, you can run ArgoCD locally with these simple steps:
```
Comment (jannfis) [inline docs/developer-guide/running-locally.md:9]: Same as above :point_up:

**Review by jannfis — CHANGES_REQUESTED** [review]
Good catch, thank you very much! Please note my review comments.

**Comment by shuheiktgw** [comment]
@jannfis Thank you for your comment! I've fixed it!

**Review by jannfis — APPROVED** [review]
Thank you very much, @shuheiktgw!

**Pull request merged**